src/core/containers/ServerHtml.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom/server';
+import serialize from 'serialize-javascript';
+import Helmet from 'react-helmet';
+
+
+export default class ServerHtml extends Component {
+
+  static propTypes = {
+    appName: PropTypes.string.isRequired,
+    assets: PropTypes.object.isRequired,
+    component: PropTypes.object.isRequired,
+    env: PropTypes.string.isRequired,
+    sriData: PropTypes.object,
+    store: PropTypes.object.isRequired,
+  };
+
+  getStatic({filePath, type} = {}) {
+    const { env, sriData, appName } = this.props;
+    const leafName = filePath.split('/').pop();
+    const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
+    let sriProps = {};
+    // Only output files for the current app.
+    if (leafName.startsWith(appName)) {
```

Comment:
This feels like it should be outside of this component to me. Probably `isDeployed` too. That would mean this helper could likely go away and just the SRI part could be shared:

```jsx
<ServerHtml styles={filterForApp(appName, assets.style)} includeSri={isDeployed} />

// ...
getStyles() {
  return this.props.styles.map((style) =>
    <link rel="stylesheet" href={style} {...this.sriProps(style)} />);
}

sriProps(filePath) {
  const { includeSri, sriData } = this.props;
  if (!includeSri) {
    return {};
  }
  const sriProps = {
    integrity: sriData[filePath.split('/').pop()],
    crossOrigin: 'anonymous',
  };
  if (!sriProps.integrity) {
    throw new Error(...);
  }
  return sriProps;
}